sensors.yaml

```diff
@@ -193,17 +193,18 @@ sensors:
         name_english: "Steering Angle Sensor"
         analog_subtype: Voltage
         points:
-            - (0.33, 0) # 0.5 * 3.3/5
-            - (2.97, 360) # 4.5 * 3.3/5
+            - (0.77213, -105)
+            - (2.71541, 105)
+            # -105, 60.29 -> 0.1674722 -> 0.77213V
+            # 105, 325.283203 -> 2.715410V
         
     exhaust_temp_sensor:
         # Sensata HTS_0630_330
         name_english: "Exhaust Gas Temp Sensor"
         analog_subtype: Resistive
         points:
-            - (785.7, -40) # datasheet says it draws 2.8mA from 5V and a 1KOhm pull up
             - (190.4, 950) # datasheet says it draws 4.2mA from 5V and a 1KOhm pull up
-
+            - (785.7, -40) # datasheet says it draws 2.8mA from 5V and a 1KOhm pull up

```

Comment:
You were correct in your commit message, you have to change the value in perentheses